draft-pignataro-eimpact-icmp.xml

```diff
@@ -458,7 +458,48 @@ The "Year" field must contain the year in which the certificate was issued, or 0
 
     <section title="Sample Use Case">
       <t>
-        This section explains a typical use case of the extensions.
+        The ICMP extension defined in this memo can be used to get sustainability metrics, either via a
+        "transactional" or a "scheduled automated" fashion, inside one administrative domain.
+        Mentioned in this section is a sample "transactional" use case of how the metrics will be
+        displayed to the user. As traceroute or ping utilities are based on ICMP, we will use
+        traceroute as an example.
+      </t>
+      <t>  
+        The flavour of traceroute that has been shown here, does support the ICMP extensions and is
+        capable of deliverintg the metrics in the extensions to the end user along with the nodes that 
+        the original datagram visited on it's way to the destination. And, it also is backwards-compatible,
+        i.e., it can also work well with those nodes which doesn't support ICMP extensions.
+      </t>
+      <t>
+        Host A executes a traceroute to 192.168.2.1 and the <xref target="use-case-1" /> shows how the user
+        of Host A will be presented with the metrics:
+        <figure anchor="use-case-1" title="Sustainability Metrics infused Traceroute">
+          <artwork xml:space="preserve" >
+> traceroute 192.168.2.1
+Tracing route to 192.168.2.1 over a maximum of 30 hops
+
+  1     3 ms     1 ms     2 ms  192.168.2.120
+       Power(Node=160W,Fan=7W,Chassis=10W)
+       Throughput(53687091200bps)
+       EERC(ISO 14001:2015, Energy-efficient ethernet)
+
+  2    12 ms     9 ms     9 ms  192.168.2.124
+       Power(Node=163W,Fan=8W,Chassis=11W)
+       Throughput(55834574848bps)
+       EERC(ISO 14001:2015)
+
+  3    12 ms     9 ms     9 ms  192.168.2.123
+
+  4     7 ms     4 ms     7 ms  192.168.2.122
+       Power(Node=150W,Fan=6W,Chassis=9W)
+       Throughput(51539607552bps)
+       EERC(ISO 14001:2015, Energy-efficient ethernet)
+
+  5     4 ms     3 ms     3 ms  192.168.2.1
```

Comment:
I think it's odd that there's no power, throughput, eerc information available for this node?  If you intend to show that this information can also be unavailable, then I'd rather skip it for a node in the middle.

Reply:
Yeah, so my initial thoughts about not including the extension information in the 5th hop was to show that there can be cases where node(s) in a network won't understand extensions/won't have extensions enabled.

So, to make it clear to the reader, should I add an extra line or so to support the above mentioned thought process or just remove the line itself?

Reply:
Ahhhhh never mind!
I was about to say that my point is that skipping it for the last node looks like a special case, and it would be better to skip the new information for a node in the middle somewhere.

BUT somehow I managed to overlook node 3 !   This one also doesn't have the information. Combined, it looks good as it is - please ignore my comment, this is all fine then, in my opinion!